### PR TITLE
HUE-9614 [Deprecation Warning] on_delete will be a required arg for ForeignKey in Django 2.0

### PR DESCRIPTION
**Pull request by agl29**

### CODE REVIEW

File: apps/beeswax/src/beeswax/models.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ class STATE(Enum):
                  (librdbms_dbms.MYSQL, 'MySQL'), (librdbms_dbms.POSTGRESQL, 'PostgreSQL'),
                  (librdbms_dbms.SQLITE, 'sqlite'), (librdbms_dbms.ORACLE, 'oracle'))
 
-  owner = models.ForeignKey(User, db_index=True)
+  owner = models.ForeignKey(User, on_delete=models.CASCADE, db_index=True)
```
Comment (romainr) [inline apps/beeswax/src/beeswax/models.py:67]: We could need to generate new db migration files if we updates the models.

Also, could you explain what was the warning, do we need to be explicit now? Was the default not 'CASCADE'?

**Review by romainr — COMMENTED** [review]
Like discussed previously could you explain the list of warnings fixed and send via multiple commits?

File: apps/beeswax/src/beeswax/models.py (hunk 1 above)
Comment (agl29) [reply apps/beeswax/src/beeswax/models.py:67]: warning is `on_delete will be a required arg for ForeignKey in Django 2.0. Set it to
models.CASCADE on models`
and the link is https://docs.djangoproject.com/en/2.0/ref/models/fields/#django.db.models.ForeignKey.on_delete

**Comment by romainr** [comment]
Next time feel free to put this type of comment https://github.com/cloudera/hue/pull/1461#discussion_r538669563 directly in the commit description!

For the DB migration, does `./build/env/bin/hue makemigrations` creates new files?

https://docs.djangoproject.com/en/3.1/topics/migrations/

**Comment by agl29** [comment]
`./build/env/bin/hue makemigrations` showing **No changes detected**

**Review by romainr — APPROVED** [review]

**Pull request merged**